src/platform-apps/channels/channel-list.test.tsx

```diff
@@ -36,10 +37,10 @@ describe('ChannelList', () => {
 
   it('renders Link to channel id', () => {
     const channels = [
-      { id: 'one', name: 'first channel' },
-      { id: 'two', name: 'second channel' },
-      { id: 'three', name: 'third channel' },
-      { id: 'four', name: 'fourth channel' },
+      { id: 'one', name: 'first channel', category: 'catg1' },
+      { id: 'two', name: 'second channel', category: 'catg1' },
+      { id: 'three', name: 'third channel', category: 'catg2' },
+      { id: 'four', name: 'fourth channel', category: 'catg2' },
```

Comment:
same as above. there's no reason to have categories on the channels in this test.